src/common/transport.cc

```diff
@@ -139,11 +139,7 @@ void Transport::handleIncoming(const std::shared_ptr<Peer> &peer) {
           handler_->onInput(buffer, totalBytes, peer);
         }
       } else {
-        if (errno == ECONNRESET) {
           handlePeerDisconnection(peer);
```

Comment:
(minor nit): Please adjust indent level on "handlePeerDisconnection()", otherwise, I'll have to clean it up the next time I run "clang-tidy" on the code.

Reply:
I adjusted it ,please checkout again